client/signup/navigation-link/test/index.jsx

```diff
@@ -45,42 +50,43 @@ describe( 'NavigationLink', () => {
 	} );
 
 	test( 'should render Button element', () => {
-		const wrapper = shallow( <NavigationLink { ...props } /> );
+		const { container } = render( <NavigationLink { ...props } /> );
 
-		expect( wrapper.find( 'ForwardRef(Button)' ) ).toHaveLength( 1 );
-		expect( wrapper.find( 'ForwardRef(Button)' ).props().className ).toEqual( 'navigation-link' );
+		expect( screen.queryAllByRole( 'button' ) ).toHaveLength( 1 );
```

Comment:
I think we meant this instead:

```suggestion
		expect( screen.getByRole( 'button' ) ).toBeVisible();
```

Because I don't see any value to query for multiple buttons here.